lib/store.js

```diff
@@ -62,11 +64,38 @@ class Store {
       ? this.player1.name
       : this.player2.name;
   }
+  sendNextPlayerToBrowser() {
```

Comment:
this becomes private

(put a _ before the name like `_sendNextPlayerToBrowser()`)